### PR TITLE
Load animations stored in Asset Catalogs

### PR DESCRIPTION
**Pull request by fouquet**
We're using a lot of Lottie animations in our iOS app. All of them are stored in the Asset Catalogs because it's just cleaner and more maintainable than to just keep them in the file system.

Unfortunately Lottie 3 removed the ability to load animations from Asset Catalogs, so I've added an additional initialiser and a loading method to enable this feature again.

### CODE REVIEW

File: lottie-swift/src/Public/iOS/AnimationFromAssetLoading.swift [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+//  AnimationFromAssetLoading.swift
+//  Lottie
+//
+//  Created by Fouquet, René on 22.07.19.
+//
+
+import Foundation
+import UIKit
+
+public extension AnimationView {
```
Comment (buba447) [inline lottie-swift/src/Public/iOS/AnimationFromAssetLoading.swift]: Would be nice if this solution supported MacOS as well.

These extensions could be added to `AnimationPublic.Swift` and `AnimationViewInitializers.swift` with an additional wrapper added around the Asset catalog stuff.

**Comment by buba447** [comment]
@thejohnlima see my earlier comment.

**Comment by fouquet** [comment]
It's on my todo list, but this list is long :(

**Comment by jouni-kivra** [comment]
Cannot find this in `Lottie 3.2.1` release for iOS, but the code still works just fine

**Comment by fouquet** [comment]
@buba447 Took me a while, but I finally got around it. I hope this is to your satisfaction now :-)

**Comment by fouquet** [comment]
@buba447 Hey, did you have a chance to look at this yet? It would be nice to get rid of our local fork and use the official Lottie pod again ;-)

**Comment by calda** [comment]
@fouquet, This seems like a nice addition -- would be happy to merge once this PR is rebased and merge conflicts are resolved. 

Closing this for now since the change has gone stale, but feel free to re-open once/if it's ready for another look.

**Pull request closed without merging**